### PR TITLE
fix install fsck utilities

### LINKED ISSUE #1515: fsck utilities are not installed
**Describe the bug**
/sbin/fcsk.* and /usr/sbin/fsck.* are not installed inside initramf not matter if hostonly or not.

When i run dracut --force --keep from 053 version all the fsck.* utilities are installed

**Distribution used**
OpenMandriva Lx Cooker

**Dracut version**
054

**Init system**
systemd

**To Reproduce**
dracut --force --keep

**Expected behavior**
/sbin/fsck.* and /usr/sbin/fsck.* utilities are installed

**Additional context**
here is a PR that fixes this situation
https://github.com/dracutdevs/dracut/pull/1514

### PR DESCRIPTION
**Pull request by tpgxyz**
This pull request fixes the installation of fsck programs which got broken by this commit https://github.com/dracutdevs/dracut/commit/82d9fbf193b0d8bd810c6cbee726a1f851e5ee75#diff-52b92d069b8698c8b6332b0357d1ffe1bec3f07f4a23406a3c697f11d2cb10ce

## Checklist
- [x] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #1515

### CODE REVIEW

**Review by haraldh — APPROVED** [review]

**Comment by tpgxyz** [comment]
@haraldh @danimo @johannbg Guys i think this is quite important. Bu releaseing 054 many users after update and initrd regeneration may end up with not bootable system, because of missing fsck utilities. I guess some may handle that incovience, but i'd suggest to merge this PR and release new version of dracut.

**Review by johannbg — CHANGES_REQUESTED** [review]
You need to fix your commit msg
from
`fix install fsck utilities`
to
fix(fs-lib): install fsck utilities

**Comment by tpgxyz** [comment]
@johannbg done

File: modules.d/99fs-lib/module-setup.sh [hunk 1]
```diff
@@ -84,5 +84,5 @@ install() {
         inst_simple /etc/e2fsck.conf
     fi
 
-    inst_multiple -o "$_helpers" fsck
+    inst_multiple -o $_helpers fsck
```
Comment (haraldh) [inline modules.d/99fs-lib/module-setup.sh:87]:
```suggestion
    # shellcheck disable=SC2086
    inst_multiple -o $_helpers fsck
```

**Review by haraldh — CHANGES_REQUESTED** [review]

**Comment by haraldh** [comment]
I even would suggest this version:
```bash
# called by dracut
install() {
    local _helpers

    inst "$moddir/fs-lib.sh" "/lib/fs-lib.sh"
    : > "${initdir}"/etc/fstab.empty

    [[ $nofscks == "yes" ]] && return

    if [[ $fscks == "${fscks#*[^ ]*}" ]]; then
        _helpers=(
            /sbin/fsck* /usr/sbin/fsck*
            xfs_db xfs_check xfs_repair xfs_metadump
            e2fsck jfs_fsck reiserfsck btrfsck
        )
        if [[ $hostonly ]]; then
            read -r -a _helpers < <(for_each_host_dev_fs echo_fs_helper)
        fi
    else
        read -r -a _helpers <<< "$fscks"
    fi

    _helpers+=(umount mount)

    if [[ ${_helpers[*]} == *e2fsck* ]] && [[ -e $dracutsysrootdir/etc/e2fsck.conf ]]; then
        inst_simple /etc/e2fsck.conf
    fi

    inst_multiple -o "${_helpers[@]}" fsck
}
```

**Comment by tpgxyz** [comment]
>     inst_multiple -o "${_helpers[@]}" fsck

Thanks for the hint... how could i forget about using it :<

**Review by haraldh — APPROVED** [review]

File: modules.d/99fs-lib/module-setup.sh [hunk 2]
```diff
@@ -67,22 +67,23 @@ install() {
...
+            read -r -a _helpers < <(for_each_host_dev_fs echo_fs_helper)
         fi
     else
-        _helpers="$fscks"
+        read -r -a _helpers <<< "$fscks"
     fi
 
+    _helpers+=(umount mount)
+
     if [[ $_helpers == *e2fsck* ]] && [[ -e $dracutsysrootdir/etc/e2fsck.conf ]]; then
```
Comment (haraldh) [inline modules.d/99fs-lib/module-setup.sh:84]:
```suggestion
    if [[ ${_helpers[*]} == *e2fsck* ]] && [[ -e $dracutsysrootdir/etc/e2fsck.conf ]]; then
```

**Review by haraldh — CHANGES_REQUESTED** [review]

File: modules.d/99fs-lib/module-setup.sh (hunk 2 above)
Comment (tpgxyz) [reply modules.d/99fs-lib/module-setup.sh]: fixed

**Review by johannbg — APPROVED** [review]

**Review by haraldh — APPROVED** [review]

**Pull request merged**